src/interfaces.jl

```diff
@@ -228,11 +228,14 @@ function connect_plot!(scene::SceneLike, plot::Combined{F}) where {F}
             transform!(t, t_user)
             plot.transformation = t
         end
-        connect!(transformation(scene), transformation(plot))
+        if is_space_compatible(plot, parent)
+            connect!(transformation(parent), transformation(plot))
+        end
```

Comment:
This only connects transformations of plots and parent plots/scenes if they have compatible spaces. For plots this checks `get(plot, :space, :data)` and for scenes it will always assume `:data` space to be compatible, with for example `:pixel` space in addition with a pixel camera.